lib/graphql/functions/array.rb

```diff
@@ -7,17 +7,15 @@ class Array < Base
       argument :limit, types.Int
       argument :offset, types.Int
 
-      def call(_, args, _)
-        query = args[:ids] ? @model_class.where(id: args[:ids]) : @model_class.all
-        query = query.offset(args[:offset]) if args[:offset]
-        query = query.limit(args[:limit]) if args[:limit]
-        query
-      end
+      def call(*attrs)
+        _, args, = attrs
+        relation = @model_class
+          .all
+          .where(args[:ids] ? { id: args[:ids] } : nil)
```

Comment:
The `nil` is unnecessary, you can do this `where({ id: args[:ids] } if args[:ids])`